internal/database/cache.go

```diff
@@ -24,14 +22,14 @@ type Cache struct {
 	subscription map[string]*SubscriptionDocument
 }
 
-type operationCacheIterator struct {
-	operation map[string]*OperationDocument
-	err       error
+type cacheIterator struct {
```

Comment:
For my understanding, why is there an in-memory cache for CosmosDB? Was there a particular REST p95 that couldn't be reached without caching?

Reply:
It's there for historical reasons, from early in development before we had access to Cosmos DB.  It was decided to keep it around as a unit test aid, and for a time it was helpful when our database use cases were simple.  But it's technical debt now; it's become a obstacle to leveraging more advanced features of Cosmos DB.  I'm honestly ready to ditch it and just use [gomock](https://github.com/uber-go/mock) or some such for unit tests, but I'll need to find time to pay that debt.